pkg/skaffold/config/options.go

```diff
@@ -85,6 +85,7 @@ type SkaffoldOptions struct {
 	Command            string
 	RPCPort            int
 	RPCHTTPPort        int
+	BuildConcurrency   int
```

Comment:
I tried to follow the conventions of other flags hence added it here

Reply:
looks good